### ISSUE TITLE
The website opens only after restarting the router

### ISSUE THREAD

**Issue by enorenio** [dart-lang/pub-dev#4331]
As a title claim, the website can only be accessed after restarting the router.
Tried with Chrome (any version) and Edge.
They show "ERR_CONNECTION_CLOSED" error.
This bug was from the very beginning, I remember even a year ago I came in and still had to restart the router.
Tried with different routers, different PCs and even different ISPs.
Same happens not only with web-site, but with packages and flutter itself. But it does not happen to any other resource on the internet.
My dns is set to 8.8.8.8 and 8.8.4.4 which is google.
My guess is that there's something on this service that is being sent with first packages, that isn't supported by my national ISP. Or some error in dns routing. My another suggestion is that you use frequently changing ddns but my isp isn't allowing the ddns change packages to propagate to the forwarding table of routers.

Please suggest any reasonable solution or, at lease, a reason for such bug.

**Comment by isoos** [comment]
@enorenio: could you please share a bit more information about your location and ISPs?

**Comment by enorenio** [comment]
@isoos Location: Tashkent. Checked on two ISPs: "uztelecom" and "turon telecom"

**Comment by isoos** [comment]
Closing, as we can't really add much to fix this issue. If somebody still has similar issue, please run the following script and attach its output alongside a new issue:
https://github.com/dart-lang/pub-dev/blob/master/app/bin/tools/check_domain_access.dart
